navigationrail/sample/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.microsoft.device.dualscreen.sample.navigationrail.MainActivity">
+
+    <com.microsoft.device.dualscreen.navigationrail.NavigationRailView
+        android:id="@+id/nav_rail_view"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        app:headerLayout="@layout/header_view"
+        app:labelVisibilityMode="labeled"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:menu="@menu/navigation_rail_menu"
+        app:menuGravity="center" />
+
+    <FrameLayout
```

Comment:
I think we don't need this container, since the `include` one, below, is a constraintlayout already.

Reply:
That is supposed to be the container for the page UI, ex: where the fragments would go.